Engine/ResourceManagement/Importer/SoundImporter.cpp

```diff
@@ -18,6 +18,33 @@ FileData SoundImporter::ExtractData(Path& assets_file_path, const Metafile& meta
 		Path* setup_exported_folder_path = App->filesystem->MakeDirectory(WWISE_INIT_PATH);
 		setup_exported_folder_path->Save(WWISE_INIT_NAME, data);
 		return { nullptr,0 };
+
+	}	
+	std::string soundbank_events_file_path = assets_file_path.GetFullPathWithoutExtension() + std::string(".txt");
+	std::string imported_events_file_name = std::to_string(metafile.uuid) + std::string("_events");
+	std::string exported_events_directory = metafile.exported_file_path.substr(0, metafile.exported_file_path.find_last_of("/"));
+
+	FileData file_imported_data = App->filesystem->GetPath(soundbank_events_file_path)->GetFile()->Load();
+	std::string file_data;
+	file_data.assign(static_cast <const char*> (file_imported_data.buffer), file_imported_data.size);
```

Comment:
`file_imported_data` is copied to the `std::string` `file_data`, and then it's not used anymore. The thing is that the data aquired using `File` method `Load()` should be freed after usage, so there is a memory leak here.